app/Dockerfile

```diff
@@ -10,4 +10,4 @@ EXPOSE 80
 
 WORKDIR /app
 
-CMD ["/usr/local/bin/node", "/app/app.js"]
\ No newline at end of file
+CMD ["/usr/local/bin/node", "/app/app.js"]
```

Comment:
Maybe change this to just `node`? - I use this in the ID Dashboard container...so I know it works.